hyperspy/tests/misc/test_cython.py

```diff
@@ -0,0 +1,22 @@
+import nose.tools as nt
+import warnings
+
+import os
+
+my_path = os.path.dirname(__file__)
+
+def test_cythonization():
+    try:
+        import Cython
+    except ImportError:
+        warnings.warn("""no cython library were found, aborting testing of cythonization""")
+    else:
+        if not os.path.exists(my_path + '/cython/test_cython_integration.c'):
+            raise DeprecationWarning('deprecation of cythonization of pyx extensions detected!')
```

Comment:
This should probably raise an error, not a `DeprecationWarning`. Something like "Cythonization of <file path> failed"